<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OPDS Browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="rootLayout">
      <item>
       <widget class="QLineEdit" name="urlEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="mainLayout" stretch="1,4">
        <item>
         <layout class="QVBoxLayout" name="bookmarksLayout">
          <item>
           <widget class="QListView" name="bookmarksView">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="bookmarksButtonsLayout">
            <item>
             <widget class="QPushButton" name="bookmarkAdd">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bookmarkEdit">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bookmarkRemove">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="browserView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>609</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>Bookmarks</string>
    </property>
    <addaction name="actionBookmarkAdd"/>
    <addaction name="actionBookmarkEdit"/>
    <addaction name="actionBookmarkRemove"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBookmarkAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionBookmarkEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionBookmarkRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBookmarkAdd</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionBookmarkAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBookmarkEdit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionBookmarkEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBookmarkRemove</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionBookmarkRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmarkRemove</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionBookmarkRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmarkEdit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionBookmarkEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmarkAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionBookmarkAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmarksView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>actionNavigateBookmark(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browserView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>actionBrowserViewClick(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionAbout()</slot>
  <slot>actionExit()</slot>
  <slot>actionBookmarkAdd()</slot>
  <slot>actionBookmarkEdit()</slot>
  <slot>actionBookmarkRemove()</slot>
  <slot>actionNavigateBookmark(QModelIndex)</slot>
  <slot>actionBrowserViewClick(QModelIndex)</slot>
 </slots>
</ui>
